src/styles/embeds/sass/components/cart.css

```diff
@@ -40,23 +42,23 @@ $cart-item-spacing: 20px;
 
 .shopify-buy__cart {
   height: 100%;
-  background-color: $color-white;
+  background-color: var(--color-white);
   width: calc(100% - 10px);
   position: absolute;
   right: 0;
   box-shadow: -5px 0 5px rgba(0, 0, 0, 0.1);
 }
 
 .shopify-buy__cart__header {
-  padding: $gutter-cart;
-  padding-right: ($gutter-cart * 2);
+  padding: var(--gutter-cart);
+  padding-right: (var(--gutter-cart) * 2);
```

Comment:
for simple cases that we are using variables for code readability rather than needing this value to be calculated on page render. I wonder if there will be any performance impediment?